src/utils/childProcess.ts

```diff
@@ -3,7 +3,9 @@ import {
 } from 'child_process';
 import stream from 'stream';
 
-import { Log } from '@/utils/logging';
+// Removed prefix path due an error on
+// playwright/test class - see https://github.com/microsoft/playwright/issues/7121
```

Comment:
Yep.
Some `jest` users brought that up and it will be fixed adding a `moduleNameMapper` on playwright global config.
We had the same error using jest, but it was fixed adding the prefix path on jest-config.
Looking to bump playwright version on the next release and also on 1.19.0 - https://github.com/microsoft/playwright/issues/10840